lib/jblond/Diff/Renderer/Html/Merged.php

```diff
@@ -0,0 +1,241 @@
+<?php
+
+namespace jblond\Diff\Renderer\Html;
+
+use jblond\Diff\Renderer\MainRenderer;
+use jblond\Diff\Renderer\SubRendererInterface;
+
+/**
+ * Merged diff generator for PHP DiffLib.
+ *
+ * PHP version 7.2 or greater
+ *
+ * @package         jblond\Diff\Renderer\Text
+ * @author          Ferry Cools <info@DigiLive.nl>
+ * @copyright   (c) 2020 Ferry Cools
+ * @license         New BSD License http://www.opensource.org/licenses/bsd-license.php
+ * @version         2.2.1
+ * @link            https://github.com/JBlond/php-diff
+ */
+class Merged extends MainRenderer implements SubRendererInterface
+{
+    /**
+     * @var array   Associative array containing the default options available for this renderer and their default
+     *              value.
+     *              - format            Format of the texts.
+     *              - insertMarkers     Markers for inserted text.
+     *              - deleteMarkers     Markers for removed text.
+     *              - title1            Title of the 1st version of text.
+     *              - title2            Title of the 2nd version of text.
+     */
+    protected $subOptions = [
+        'format'        => 'html',
+        'insertMarkers' => ['<ins>', '</ins>'],
+        'deleteMarkers' => ['<del>', '</del>'],
+        'title1'        => 'Version1',
+        'title2'        => 'Version2',
+    ];
+    protected $lineOffset = 0;
```

Comment:
Here is the PHP doc missing

Reply:
Actually there's a bit more missing or invalid. I intended to fix all when the acutal output of the renderer was discussed.